src/helpers/EmailNotificationAPI.js

```diff
@@ -1,3 +1,4 @@
+import 'babel-polyfill';
```

Comment:
You should remove this from here and add it at the top of `src/index.js` file. That way, it will be included in every method/file that needs it.